index.html

```diff
@@ -50,12 +50,18 @@ <h4 class="summit__logo__bot">2023</h4>
 
           <!-- Navbar links -->
           <ul class="navbar__links">
-            <li><a href="#">Home</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#program">Program</a></li>
-            <li><a href="#speakers">Speakers</a></li>
-            <li><a href="#sponsors__section">Partners</a></li>
-            <li><a href="#">World Tourism Summit</a></li>
+            <li><a href="index.html" onclick="toggleMenu()">Home</a></li>
+            <li><a href="about.html" onclick="toggleMenu()">About</a></li>
+            <li><a href="#program" onclick="toggleMenu()">Program</a></li>
+            <li><a href="#speakers" onclick="toggleMenu()">Speakers</a></li>
+            <li>
+              <a href="#sponsors__section" onclick="toggleMenu()">Partners</a>
+            </li>
+            <li>
+              <a href="index.html" onclick="toggleMenu()"
+                >World Tourism Summit</a
+              >
+            </li>
           </ul>
 
```

Comment:
![Screenshot from 2023-05-25 03-09-36](https://github.com/sergio-usma/First_capstone_project/assets/83042742/9bd24386-0f30-407e-948e-515589826cc2)
- [ ] Kindly ensure that your main section description sticks as much as possible to the template design. Right now, I noticed that it misses the `white border` and the appropriate `font-size`.
### Template Design
![Screenshot from 2023-05-25 03-02-43](https://github.com/sergio-usma/First_capstone_project/assets/83042742/b43de1a2-2b0c-4b45-8947-1fd4c233a212)

### Template Design
![Screenshot from 2023-05-25 03-13-15](https://github.com/sergio-usma/First_capstone_project/assets/83042742/e0c7c6b8-aa8a-42c1-8cd4-46d5de5b75e2)

Reply:
Ok! I added the white border and the appropriate font-size.